File: 151002/Kragel/dc/src/main/java/by/bsuir/dc/repository/StickerRepository.java
package by.bsuir.dc.repository;

import by.bsuir.dc.entity.Sticker;
import by.bsuir.dc.repository.common.InMemoryCrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public class StickerRepository extends InMemoryCrudRepository<Sticker> {
}

File: 151004/Akimenko/DC_task_04/kafka/Publisher/Repository/Implementation/StickerRepository.cs
using Publisher.Entity.Db;
using Publisher.Repository.Implementation.Common;
using Publisher.Repository.Interface;
using Publisher.Storage.Common;

namespace Publisher.Repository.Implementation
{
    public class stickerRepository(DbStorage context) : AbstractCrudRepository<sticker>(context), IstickerRepository
    {
    }
}
